tests/e2e-net-rules/helpers.go

```diff
@@ -0,0 +1,547 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+func GetTraceeArgumentByName(event trace.Event, argName string) (trace.Argument, error) {
+	for _, arg := range event.Args {
+		if arg.Name == argName {
+			return arg, nil
+		}
+	}
+	return trace.Argument{}, fmt.Errorf("argument %s not found", argName)
+}
+
+func GetTraceeStringByName(event trace.Event, argName string) (string, error) {
+	for _, arg := range event.Args {
+		if arg.Name == argName {
+			argStr, ok := arg.Value.(string)
+			if ok {
+				return argStr, nil
+			}
+		}
+	}
+	return "", fmt.Errorf("argument %s not found", argName)
+}
+
+//
+// Network Protocol Event Types
```

Comment:
Since you're using the helpers already defined in `signatures/helpers`, why not add the rest of these helpers there and import them?

Reply:
I thought about it (so signatures could use them as well). Then I saw CNDR did something similar and wasn't sure. I think you are right!